src/ExpressionEvaluator/CSharp/Source/ExpressionCompiler/Microsoft.CodeAnalysis.CSharp.ExpressionCompiler.csproj

```diff
@@ -8,6 +8,7 @@
     <AssemblyName>Microsoft.CodeAnalysis.CSharp.ExpressionEvaluator.ExpressionCompiler</AssemblyName>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <ApplyNgenOptimization>full</ApplyNgenOptimization>
+    <IsPackable>true</IsPackable>
```

Comment:
Why do we need this property on so mayn new projects?

Reply:
Yes. All shipping assemblies need to be in a package.

Reply:
Arcade symbol publishing is package based. Making these packages is the simplest way to do this. Tried to construct a symbol package for some of these and it's not really possible without hacks.